.github/workflows/unit-test.yml

```
@@ -75,7 +77,7 @@ jobs:
           path: .qodana/code-coverage
 
       - name: Execute analysis
-        uses: JetBrains/qodana-action@main
+        uses: JetBrains/qodana-action@v2024.2
```

Since we are pinning its version, let's use the latest one
```suggestion
        uses: JetBrains/qodana-action@v2025.2
```